experimental/types/value_metadata.go

```diff
@@ -45,154 +45,157 @@ const (
 	NotValueMetadataUnicode
 )
 
+// MetadataStrings provides a mapping of strings to metadata for quick lookup.
+var metadataStrings = map[string]DataMetadata{
+	"numeric":          ValueMetadataNumeric,
+	"boolean":          ValueMetadataBoolean,
+	"alphanumeric":     ValueMetadataAlphanumeric,
+	"ascii":            ValueMetadataAscii,
+	"base64":           ValueMetadataBase64,
+	"uri":              ValueMetadataURI,
+	"domain":           ValueMetadataDomain,
+	"unicode":          ValueMetadataUnicode,
+	"not_numeric":      NotValueMetadataNumeric,
```

Comment:
I was thinking if we can even remove the negated ones. So have one source of truth

Reply:
Done